src/scripts/main.js

```diff
@@ -1,3 +1,317 @@
 'use strict';
 
-// write code here
+const table = document.querySelector('table');
+const tbody = table.querySelector('tbody');
+const thead = document.querySelector('thead');
+const rows = [...table.querySelectorAll('tbody tr')];
+const headerCells = thead.querySelectorAll('th');
+let isAscending = true;
+let lastClicked = null;
+
+// sorting by click
+headerCells[0].addEventListener('click', () => {
+  if (lastClicked !== 0) {
+    isAscending = true;
+    lastClicked = 0;
+  }
+
+  const sortedRows = rows.sort((rowA, rowB) => {
+    const nameA = rowA.querySelector('td:nth-child(1)').textContent.trim();
+    const nameB = rowB.querySelector('td:nth-child(1)').textContent.trim();
+
+    return isAscending
+      ? nameA.localeCompare(nameB)
+      : nameB.localeCompare(nameA);
+  });
+
+  tbody.innerHTML = '';
+
+  sortedRows.forEach((row) => tbody.appendChild(row));
+
+  isAscending = !isAscending;
+});
+
+headerCells[1].addEventListener('click', () => {
+  if (lastClicked !== 1) {
+    isAscending = true;
+    lastClicked = 1;
+  }
+
+  const sortedRows = rows.sort((rowA, rowB) => {
+    const positionA = rowA.querySelector('td:nth-child(1)').textContent.trim();
+    const positionB = rowB.querySelector('td:nth-child(1)').textContent.trim();
```

Comment:
The sorting logic for the 'Position' column is incorrect. It should be comparing the second column (td:nth-child(2)) instead of the first column (td:nth-child(1)).